tests/flytekit/unit/annotated/test_references.py

```diff
@@ -64,10 +64,13 @@ def ref_t1(a: typing.List[str]) -> str:
 
 def test_ref_task_more():
     @reference_task(
```

Comment:
if someone forgets to specify the ID and just does
```
@reference
def r1():
    ...
```
do we get an error?

Reply:
No there won't be an error if they all have default value.